build-image/webif.bash

```diff
@@ -11,7 +11,7 @@ port=80 # Port the webserver is listing to
 source /etc/openhabian.conf # to get the hostname
 
 if [ "$1" = "start" ]; then
-  mkdir /tmp/webif
+  mkdir /tmp/webif 2>/dev/null
```

Comment:
maybe  `mkdir -p /tmp/webif` is the better choice, not complaining about existing directories, and reporting severe errors

Reply:
ok